scripts/bfasst/compare_waveforms/analyze_graph.py

```diff
@@ -21,14 +21,10 @@ def analyze_graphs(path, module):
     impl_fst = path / Path(module + "_impl.vcd.fst")
```

Comment:
If `path` is already a Path object, you don't need to create another Path object.  For example, this code works and is cleaner:
```
impl_fst = path / module + "_impl.vcd.fst"
```

Reply:
@JakeEdvenson